qa/tasks/cephfs/test_volume_client.py

```diff
@@ -14,11 +14,12 @@ class TestVolumeClient(CephFSTestCase):
     # One for looking at the global filesystem, one for being
     # the VolumeClient, two for mounting the created shares
     CLIENTS_REQUIRED = 4
-    py_version = 'python'
+    default_py_version = 'python'
 
     def setUp(self):
         CephFSTestCase.setUp(self)
-        self.py_version = self.ctx.config.get('overrides', {}).get('python', 'python')
+        self.py_version = self.ctx.config.get('overrides', {}).\
+                          get('python', TestVolumeclient.default_py_version)
```

Comment:
`TestVolumeClient`

````
2019-06-15T15:38:32.002 INFO:tasks.cephfs_test_runner:Traceback (most recent call last):
2019-06-15T15:38:32.002 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-pdonnell-testing-20190614.222049/qa/tasks/cephfs/test_volume_client.py", line 22, in setUp
2019-06-15T15:38:32.002 INFO:tasks.cephfs_test_runner:    get('python', TestVolumeclient.default_py_version)
2019-06-15T15:38:32.002 INFO:tasks.cephfs_test_runner:NameError: global name 'TestVolumeclient' is not defined
````

Reply:
Geez, that's sad. I've made the change and ran a random test from test_volume_client.py to avoid it -

```
2019-06-19 18:10:48,517.517 INFO:__main__:Stopped test: test_purge (tasks.cephfs.test_volume_client.TestVolumeClient) in 34.355982s
2019-06-19 18:10:48,518.518 INFO:__main__:
2019-06-19 18:10:48,518.518 INFO:__main__:----------------------------------------------------------------------
2019-06-19 18:10:48,518.518 INFO:__main__:Ran 1 test in 34.357s
2019-06-19 18:10:48,518.518 INFO:__main__:
2019-06-19 18:10:48,518.518 INFO:__main__:OK
```